_data/sidebars/community_sidebar.yaml

```diff
@@ -119,6 +119,10 @@ entries:
       url: /community-guidelines-adapters-openfoam.html
       output: web
     
+  - title: Forum Browser
+    url: /community-forum-browser.html
+    output: web
```

Comment:
A more fitting place would be in the `docs_sidebar.yaml`, under `Fundamentals`, between `Roadmap` and `Previous versions`.

A better title would be `FAQ` and initialized with the `faq` tag in all categories.